html/interaction/spelling-and-grammar-checking/references/spelling-markers-001.html

```diff
@@ -0,0 +1,7 @@
+<!DOCTYPE html>
```

Comment:
I see there's already a folder called `html/editing/editing-0/spelling-and-grammar-checking`, the path is quite strange but probably these files fit there too. Or maybe we can move the other to this new folder.

I don't know how WTP repo is organized in term of folders, so this is just a suggestion.